aytests/sles12_network.xml

```diff
@@ -638,6 +638,29 @@ ls -l /mnt/$BASE
       </rule>
     </net-udev>
   </networking>
+  <host>
```

Comment:
Used same section already defined in https://github.com/yast/aytests-tests/blob/master/aytests/first_stage_hosts.xml#L20